recipes/asdf-transform-schemas/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "asdf-transform-schemas" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asdf_transform_schemas-{{ version }}.tar.gz
+  sha256: ef0cc44859129fe8edd9946ef5d286307176e5623c2496ffd6f7120e036abfbe
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - setuptools-scm
+  run:
+    - asdf >=2.8.0
+    - importlib_resources >=3  # [py<39]
```

Comment:
Would be better to just unconditionally depend on `importlib_resources` and make the recipe noarch.

Reply:
Yes, done.